phantazm-neuron-minestom/src/main/java/com/github/phantazmnetwork/neuron/bindings/minestom/entity/GroundMinestomDescriptor.java

```diff
@@ -0,0 +1,40 @@
+package com.github.phantazmnetwork.neuron.bindings.minestom.entity;
+
+import com.github.phantazmnetwork.neuron.agent.GroundDescriptor;
+import net.minestom.server.entity.EntityType;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * A {@link MinestomDescriptor} for gravity-bound entities.
+ */
+public interface GroundMinestomDescriptor extends MinestomDescriptor, GroundDescriptor {
+    @Override
+    default float getJumpHeight() {
+        return 1F;
+    }
+
+    @Override
+    default float getFallTolerance() {
+        return 16F;
```

Comment:
doesn't particularly matter but why such the high tolerance

Reply:
should i lower it? entities won't be able to pathfind off drops greater than whatever tolerance so it's probably best not to set it too low
3 would be another reasonable value because values greater than it cause fall damage in vanilla

Reply:
I think 3 is good